source/likely.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
 
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    likely.initiate();
+    history.init();
```

Comment:
You don't like the inconsistency across this names? Yep, backward compatibility is a burden.